internal/espresso/config.go

```diff
@@ -28,11 +28,12 @@ type Espresso struct {
 
 // TestOptions represents the espresso test filter options configuration.
 type TestOptions struct {
-	NotClass   []string `yaml:"notClass,omitempty" json:"notClass"`
-	Class      []string `yaml:"class,omitempty" json:"class"`
-	Package    string   `yaml:"package,omitempty" json:"package"`
-	Size       string   `yaml:"size,omitempty" json:"size"`
-	Annotation string   `yaml:"annotation,omitempty" json:"annotation"`
+	NotClass         []string `yaml:"notClass,omitempty" json:"notClass"`
+	Class            []string `yaml:"class,omitempty" json:"class"`
+	Package          string   `yaml:"package,omitempty" json:"package"`
+	Size             string   `yaml:"size,omitempty" json:"size"`
+	Annotation       string   `yaml:"annotation,omitempty" json:"annotation"`
+	ClearPackageData bool     `yaml:"clearPackageData,omitempty" json:"clearPackageData"`
```

Comment:
Not enough. This struct is not the one being serialized to test-composer.

Reply:
Added